aspnetcore/web-api/jsonpatch.md

```diff
@@ -14,6 +14,29 @@ By [Tom Dykstra](https://github.com/tdykstra)
 
 This article explains how to handle JSON Patch requests in an ASP.NET Core web API.
 
+::: moniker range=">= aspnetcore-3.0"
+
+## Package installation
+
+Support for JsonPatch is enabled using the `Microsoft.AspNetCore.Mvc.NewtonsoftJson` package. To enable this feature, apps must:
+
+* Install the [Microsoft.AspNetCore.Mvc.NewtonsoftJson](https://www.nuget.org/packages/Microsoft.AspNetCore.Mvc.NewtonsoftJson/) NuGet package.
+* Update the project's `Startup.ConfigureServices` method to include a call to `AddRazorRuntimeCompilation`:
+
+  ```csharp
+  services
+      .AddControllersWithViews()
+      .AddNewtonsoftJson();
+  ```
+
+`AddNewtonsoftJson` is compatible with the new MVC service registration methods:
```

Comment:
```suggestion
`AddNewtonsoftJson` is compatible with the MVC service registration methods:
```

Reply:
I copied this text from https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-2.2&tabs=visual-studio#jsonnet-support. Does that need to change too?